.github/workflows/install_firedrake_test.yml

```diff
@@ -0,0 +1,86 @@
+name: 'Install Firedrake and Mesh Adaptation software'
+
+on:
+  # Trigger when this or any file in the install directory changes
+  pull_request:
+    paths:
+      - 'install/**'
+      - '.github/workflows/install_firedrake_test.yml'
+
+  # Trigger the workflow every Saturday at 1AM
+  schedule:
+    - cron: '0 1 * * 6'
```

Comment:
I'd like this one to be triggered less often. Say once a month.

Reply:
Changed in https://github.com/mesh-adaptation/docs/pull/77/commits/c3cb49f2c0577840e1f5c079cf28ddbf873caa92 to be run on the 1st day of each month